src/main/java/org/folio/innreach/domain/service/impl/RequestServiceImpl.java

```diff
@@ -257,6 +257,14 @@ public RequestDTO findRequest(UUID requestId) {
       "No request found with id = " + requestId));
   }
 
+  @Override
+  public void deleteRequest(UUID requestId) {
+    circulationClient.findRequest(requestId)
+      .ifPresentOrElse(requestDTO -> circulationClient.deleteRequest(requestId)
+        ,()-> log.info("Request not present with requestid:"+requestId));
```

Comment:
Same

Reply:
Same as above. Please place the comma on the previous line on all of these, and add spaces around the `->` and the `+`.